api/users-router.js

```diff
@@ -14,7 +14,17 @@ router.get('/', async (req, res) => {
 router.post('/', async (req, res) => {
   const { userEmail, fullName, admin } = req.body;
   try {
-    await addAuthorizedUser(userEmail, fullName, admin);
+    await addOrUpdateAuthorizedUser(userEmail, fullName, admin);
+    return res.sendStatus(200);
+  } catch (err) {
+    return res.sendStatus(500).json(err);
+  }
+});
+
+router.delete('/:userEmail', async (req, res) => {
```

Comment:
This is making me think we should put some extra thought into how we handle cross-referencing tagged users since there's a chance their user record could have been deleted.